xml/System/Nullable`1.xml

```diff
@@ -536,9 +536,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the `value` parameter is not `null`, the <xref:System.Nullable%601.Value%2A> property of the new <xref:System.Nullable%601> value is initialized to the `value` parameter and the <xref:System.Nullable%601.HasValue%2A> property is initialized to `true`.  
-  
- If the `value` parameter is `null`, the <xref:System.Nullable%601.Value%2A> property of the new <xref:System.Nullable%601> value is initialized to the default value, which is the value that is all binary zeroes, and the <xref:System.Nullable%601.HasValue%2A> property is initialized to `false`.  
+ The <xref:System.Nullable%601.Value%2A> property of the new <xref:System.Nullable%601> value is initialized to the `value` parameter and the <xref:System.Nullable%601.HasValue%2A> property is initialized to `true`.  
```

Comment:
While it's true that this defined implicit operator accepts a `T` and not a `T?`, it's also true that the compiler let's you assign null to a `Nullable<T>`, in which case it generates the equivalent of setting `HasValue` to `false` and `Value` to `default`.  From that perspective, what was written was correct.  So I think the question here is whether the docs should be from the perspective of the actual raw API or the syntax the compiler enables; I'd be inclined to stick with what was there, since implicit operators are all about being used by the compiler.  But I don't have a strong opinion; if you want to go with strictly what the API declares, then the new version looks fine.

Reply:
I think that if you wanted to describe how the C# compiler converts a null literal to `T?`, it should be done in a separate paragraph that emphasizes how this implicit operator is *not* involved.